playbook/buildfarmer/test_regressions_investigation.md

```diff
@@ -45,8 +45,8 @@ To investigate failing tests, you should follow these steps:
   * If the test is failing for a long time, it might be a flaky test. [`calculate_flakiness_jobs.sql`](./buildfarmer_triage_tools.md#calculate_flakiness_jobssql) is a script that shows the failure percentage of a list of jobs that have failed at least once in the last *X* days with a specified test failure (e.g., `./sql_run.sh calculate_flakiness_jobs.sql "Stopwatch.StartStopReset" "15 days"` will show the failure percentage of the last 15 days of the test `Stopwatch.StartStopReset`).
 
 After all these steps, you should have enough information to report the issue in the package repository. If you are not sure about the reason for the failure, you should ask ROS and Gazebo teams if they know about the issue.
-* For ROS, you can ask in `ros-core-developers` Slack channel or Chris Lalancette (clalancette) in GitHub.
-* For Gazebo, you can ask in `gazebo-core-developers` Slack channel or Addisu Taddese (azeey) in GitHub.
+* For ROS, you can ask Chris Lalancette (clalancette) in GitHub.
+* For Gazebo, you can ask Addisu Taddese (azeey) in GitHub.
 
```

Comment:
Same here, probably report to the buildfarmers team could be a thing so we can triage it?